build.sh

```diff
@@ -5,7 +5,7 @@ PYTHON=${PYTHON:-python}
 usage()
 {
     echo "Usage: $0 [BuildArch] [BuildType] [clean] [verbose] [coverage] [cross] [clangx.y] [ninja] [configureonly] [skipconfigure] [skipnative] [skipmscorlib] [skiptests] [cmakeargs]"
-    echo "BuildArch can be: x64, x86, arm, arm64"
+    echo "BuildArch can be: x64, x86, arm, arm64, arm-softfp"
```

Comment:
@kvochko please can you place arm-softfp before arm here as was requested before for cross/build-rootfs.sh